src/ndlbulkdownload/cli.py

```diff
@@ -43,6 +43,21 @@
 
 failed_urls = []
 
+# Global variable to store the output directory
+OUTPUT_DIR = None
+
+
+def check_and_set_output_dir(args):
+    global OUTPUT_DIR
+    if not os.path.exists(args.output_dir):
+        # Create the directory if it doesn't exist
+        os.makedirs(args.output_dir)
+        logging.info(f"Created directory: {args.output_dir}")
+    else:
+        logging.info(f"Directory exists: {args.output_dir}")
+
+    OUTPUT_DIR = args.output_dir
```

Comment:
Since there's a global being set, may as well remove the logging info after this statement to show the final path once rather than adding branching logic.